### PR TITLE
Creation Model book

### PR DESCRIPTION
**Pull request by AleFalcon**
## Summary
* Creation model book
* Creation a book

## Trello Card

https://trello.com/c/bFZMCIDy/18-crear-modelo-de-book

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```
Comment (nicozar95) [inline app/models/book.rb]: You should validate the required fields in the Book model:
```
validates :genre, :author, :image, :title, :editor, :year, presence: true
```

Comment (AleFalcon) [reply app/models/book.rb]: This validation is on the branch rspect+testBook. Are you want that i replicate the changes?

Comment (nicozar95) [reply app/models/book.rb]: Dismiss the validation on that branch, cause that branch is for specs.
Make the validations in this branch

Comment (AleFalcon) [reply app/models/book.rb]: modified

**Pull request merged**